packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -39,7 +41,17 @@ export default class ConfigParser {
         private _moduleRequireService: ModuleRequireService = new RequireLibrary()
     ) {}
 
-    autoCompile() {
+    async autoCompile() {
+        /**
+         * Checks tsconfig.json path, throws error if it doesn't exist
+         */
+        if (this._config.autoCompileOpts?.tsNodeOpts?.project) {
+            const tsconfigPath = path.resolve(this._config.autoCompileOpts.tsNodeOpts.project)
+            if (!canAccess(tsconfigPath)) {
```

Comment:
```suggestion
            if (!(await canAccess(tsconfigPath))) {
```

Reply:
`canAccess` internally uses `fs.accessSync`. 
do we need await in here ?

Reply:
Ah! Then we don't, thanks for checking.